src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -621,6 +654,16 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 		}
 	}
 
+	/**
+	 * Returns true if the checkbox column is used for row selection in the table
+	 */
+	private isCheckboxColumnsUsedForSelection(): boolean {
+		return this.columns.some(c => {
+			const checkboxAction = <ActionOnCheck>(c.options ? (<any>c.options).actionOnCheckbox : c.action);
```

Comment:
You should avoid using any - is there not a relevant type you can use here instead?